src/Core/Batch/SysvConfigStorage.php

```diff
@@ -79,16 +79,24 @@ public function save(BatchConfig $config)
                 'Failed to attach to the shared memory'
             );
         }
-        $result = shm_put_var($shmid, self::VAR_KEY, $config);
-        shm_detach($shmid);
+
+        // If the variable write fails, clear the memory and re-raise the exception
+        try {
```

Comment:
It's currently throwing an `\Exception` on the serialization error, but if there's some other issue, the documentation says it returns `false`.